2022-03-01 12:36:27,208 INFO    MainThread:32126 [wandb_setup.py:_flush():75] Loading settings from /opt/ml/.config/wandb/settings
2022-03-01 12:36:27,209 INFO    MainThread:32126 [wandb_setup.py:_flush():75] Loading settings from /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/settings
2022-03-01 12:36:27,209 INFO    MainThread:32126 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-01 12:36:27,209 INFO    MainThread:32126 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-01 12:36:27,209 INFO    MainThread:32126 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-03-01 12:36:27,209 INFO    MainThread:32126 [wandb_init.py:_log_setup():386] Logging user logs to /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/run-20220301_123627-3sedgzpj/logs/debug.log
2022-03-01 12:36:27,209 INFO    MainThread:32126 [wandb_init.py:_log_setup():387] Logging internal logs to /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/run-20220301_123627-3sedgzpj/logs/debug-internal.log
2022-03-01 12:36:27,209 INFO    MainThread:32126 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f94684622e0>
2022-03-01 12:36:27,209 INFO    MainThread:32126 [wandb_init.py:init():420] calling init triggers
2022-03-01 12:36:27,209 INFO    MainThread:32126 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2022-03-01 12:36:27,209 INFO    MainThread:32126 [wandb_init.py:init():471] starting backend
2022-03-01 12:36:27,209 INFO    MainThread:32126 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-01 12:36:27,243 INFO    MainThread:32126 [backend.py:ensure_launched():219] starting backend process...
2022-03-01 12:36:27,269 INFO    MainThread:32126 [backend.py:ensure_launched():224] started backend process with pid: 32396
2022-03-01 12:36:27,272 INFO    MainThread:32126 [wandb_init.py:init():480] backend started and connected
2022-03-01 12:36:27,293 INFO    MainThread:32126 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-03-01 12:36:27,293 INFO    MainThread:32126 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-01 12:36:27,293 INFO    MainThread:32126 [wandb_init.py:init():550] updated telemetry
2022-03-01 12:36:27,329 INFO    MainThread:32126 [wandb_init.py:init():581] communicating current version
2022-03-01 12:36:28,123 INFO    MainThread:32126 [wandb_init.py:init():586] got version response 
2022-03-01 12:36:28,123 INFO    MainThread:32126 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-01 12:36:28,372 INFO    MainThread:32126 [wandb_init.py:init():624] starting run threads in backend
2022-03-01 12:36:31,915 INFO    MainThread:32126 [wandb_run.py:_console_start():1827] atexit reg
2022-03-01 12:36:31,915 INFO    MainThread:32126 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-03-01 12:36:31,916 INFO    MainThread:32126 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-03-01 12:36:31,919 INFO    MainThread:32126 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-01 12:36:31,919 INFO    MainThread:32126 [wandb_init.py:init():651] run started, returning control to user process
2022-03-01 12:36:31,922 INFO    MainThread:32126 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-01 12:36:31,922 INFO    MainThread:32126 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-01 12:38:58,677 INFO    MainThread:32126 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-01 12:38:58,678 INFO    MainThread:32126 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-01 12:38:58,678 INFO    MainThread:32126 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-01 12:42:42,066 INFO    MainThread:32126 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-03-01 12:42:42,069 INFO    MainThread:32126 [wandb_run.py:_restore():1769] restore
2022-03-01 12:42:42,340 INFO    MainThread:32126 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-03-01 12:42:42,571 INFO    MainThread:32126 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-03-01 12:42:43,523 INFO    MainThread:32126 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 3628
}

2022-03-01 12:42:43,627 INFO    MainThread:32126 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 3954
}

2022-03-01 12:42:43,730 INFO    MainThread:32126 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 3954
}

2022-03-01 12:42:43,833 INFO    MainThread:32126 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3954
  total_bytes: 3954
}

2022-03-01 12:42:43,935 INFO    MainThread:32126 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3954
  total_bytes: 3954
}

2022-03-01 12:42:44,037 INFO    MainThread:32126 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3954
  total_bytes: 3954
}

2022-03-01 12:42:44,139 INFO    MainThread:32126 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3954
  total_bytes: 3954
}

2022-03-01 12:42:44,241 INFO    MainThread:32126 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3954
  total_bytes: 3954
}

2022-03-01 12:42:44,343 INFO    MainThread:32126 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3954
  total_bytes: 3954
}

